test/docs-json/custom-readme-output/components/my-component/readme.md

```diff
@@ -0,0 +1,31 @@
+This file is in a custom location, set with `.dir` on the `docs-readme` OT.
```

Comment:
I feel like I may have asked this in the past, but does anything in CI run these tests in `test/docs-json/`? I'm not seeing anything other than `build.docs-json` and `build` (that calls `build.docs-json`) in `test/package.json`, but that's not always the easiest thing to track down between `cd` calls and searching for `npm run build`. I would have thought I've asked this in https://github.com/ionic-team/stencil/pull/4212/ where we create most (all?) of files in this `/test/docs-json/` dir, but couldn't find anything

Reply:
there's no tests _per se_, but the `build.docs-json` command is called by the `build` command in `test/` which I think is in turn called by the `test-analysis` command which then in turn runs in the `test-analysis.yml` workflow

https://github.com/ionic-team/stencil/blob/b569ea7a9ad5d3b43eba3cf5c556cfab2c5e9e5f/.github/workflows/test-analysis.yml#L42-L47

the 'test' is basically that things are written to the disk and will fail the dirty git context check if the output has differed

Reply:
That kind of tracks - can you adding something to the backlog to make this it's own workflow? I'm scared if we remove/change the analysis stuff, we won't realize what we've done there